### PR TITLE
Added return type on method

### PR DESCRIPTION
**Pull request by NishaJangid**
### Description (*)
Added return type on Wishlist module's method

### Fixed Issues (if relevant)

1. N/A

### Manual testing scenarios (*)

1. N/A

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [X] All commits are accompanied by meaningful commit messages
 - [X] All new or changed code is covered with unit/integration tests (if applicable)
 - [X] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @NishaJangid. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=21742) <br/>All committers have signed the CLA.

File: app/code/Magento/Wishlist/Block/Adminhtml/WishlistTab.php [hunk 1]
```diff
@@ -42,7 +42,7 @@ public function __construct(Context $context, Registry $registry, array $data =
     }
 
     /**
-     * @inheritdoc
```
Comment (VladimirZaets) [inline app/code/Magento/Wishlist/Block/Adminhtml/WishlistTab.php:45]: Hi @NishaJangid. Thanks for collaboration. ```@inheritdoc``` annotation means that function already declared in parent scope or interface and you can see the method annotation theirs. It means that not needed to declare type annotation directly for this method.

**Comment by sidolov** [comment]
@NishaJangid , I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for the collaboration!

**Pull request closed without merging**